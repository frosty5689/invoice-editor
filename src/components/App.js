import React from 'react';
import logo from '../assets/logo.svg';
import '../css/App.css';

import Invoice from './Invoice';

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <div className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h3>Welcome to Invoice Editor</h3>
        </div>
        <Invoice />
      </div>
    );
  }
}

export default App;
