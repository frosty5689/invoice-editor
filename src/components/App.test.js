import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

jest.mock('./Invoice', () => {
  const Invoice = () => <div />;
  return Invoice;
});

describe('App component', () => {

  it('App should render correctly', () => {
    const component = renderer.create(
      <App />,
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});