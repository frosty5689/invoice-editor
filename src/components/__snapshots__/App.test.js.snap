// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component App should render correctly 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h3>
      Welcome to Invoice Editor
    </h3>
  </div>
  <div />
</div>
`;
